debian/xenial/kubeadm/debian/rules

```diff
@@ -11,7 +11,7 @@ binary:
 	#curl  --fail -sS -L \
 	#	-o usr/bin/kubeadm \
 	#	"https://storage.googleapis.com/kubernetes-release/release/v{{ .Version }}/bin/linux/{{ .Arch }}/kubeadm"
-	printf '#!/bin/bash\necho "this is kubeadm"' > usr/bin/kubeadm
+	cat kubeadm-e7c8aa5.base64 | base64 --decode > usr/bin/kubeadm
```

Comment:
Yeah, it's just a local file (hence this is tagged WIP). I had encode it because the build script thinks all files may be templates and trips over the binaries. This will be a curl thing, when we have binaries in the right place.